workflow-dev/tpl/deis-builder-rc.yaml

```diff
@@ -32,6 +32,8 @@ spec:
               value: "2223"
             - name: BUILDER_STORAGE
               value: "{{ or (env "STORAGE_TYPE") (.storage)}}"
+            - name: "GIT_LOCK_TIMEOUT"
+              value: "10"
```

Comment:
A lock makes sure that we are not pushing concurrently to the same GIT repo untill the previous one is finished which takes of the order minutes